### ISSUE TITLE
Typings are missing the desc element for SVGs

### ISSUE THREAD

**Issue by programbo** [developit/preact#1296]
To improve the accessibility of SVG elements, the `<desc>` element should be applied. This currently errors with the latest Typescript typings.

See https://developer.mozilla.org/en-US/docs/Web/SVG/Element/desc

**Comment by programbo** [comment]
For some reason, I closed this myself. Fixed by PR https://github.com/developit/preact/pull/1297
